docs/03-15-trial-cleanup-cronjob.md

```diff
@@ -1,7 +1,7 @@
-# Trial Cleanup Job
+# Trial Cleanup CronJob
 
-Trial Cleanup Job is a Job that makes the SKR instances with the `trial` plan expire 14 days after their creation.
-Expiration means that the SKR instance is suspended and the `expired` flag is set.
+Trial Cleanup CronJob is a Job that makes the SAP BTP, Kyma runtime instances with the `trial` plan expire 14 days after their creation.
```

Comment:
I know that the naming convention has some rules, but replacing all `SKR` with SAP BTP, Kyma runtime looks ugly. Do we have a possibility to change it?

Reply:
maybe we can replace `SKR` with `Kyma runtime`

Reply:
I tried to use "SAP BTP, Kyma runtime" only once per doc (the first time) and then kept using "Kyma runtime".